crates/api_models/src/payouts.rs

```diff
@@ -464,6 +464,10 @@ pub struct PayoutCreateResponse {
     #[schema(value_type = Option<Object>, example = r#"{ "udf1": "some-value", "udf2": "some-value" }"#)]
     pub metadata: Option<pii::SecretSerdeValue>,
 
+    /// Connector_id related to merchant
+    #[schema(value_type = Option<String>, example = "mca_sAD3OZLATetvjLOYhUSy")]
+    pub merchant_connector_id: Option<String>,
```

Comment:
Can we have the documentation comment along the lines of "Identifier of the merchant connector account" maybe?